### PR TITLE
Add some favourite gags

### PR DESCRIPTION
**Pull request by jezen**
These always made me chuckle.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -4,6 +4,28 @@ submit your own, if they make me laugh I'll merge them.
...
+
+**Q:** How many Prolog programmers does it take to change a lightbulb?
+
+**A:** Yes.
+
+---
+
+I’ve been hearing news about this big boolean.
+
+Huge if true.
```
Comment (styfle) [inline readme.md]: This is not Q/A format

File: readme.md [hunk 2]
```diff
@@ -4,6 +4,28 @@ submit your own, if they make me laugh I'll merge them.
...
+fortnight
+
+[Error: Password is Two Week]
+```
+
+---
+
+**Q:** How many Prolog programmers does it take to change a lightbulb?
+
+**A:** Yes.
```
Comment (styfle) [inline readme.md]: You'll have to explain this one to me 😄

File: readme.md [hunk 3]
```diff
@@ -4,6 +4,28 @@ submit your own, if they make me laugh I'll merge them.
 
 ---
 
+```
+[Please Enter New Password]
+
+fortnight
+
+[Error: Password is Two Week]
```
[the remark this answers is not in the document]
Comment (jezen) [reply readme.md]: I wasn't aware it had to be. And also, some of the jokes in the repo are ruined by being forced into a Q/A format when they don't naturally fit that.

File: readme.md (hunk 2 above)
Comment (jezen) [reply readme.md]: [Prolog](https://en.wikipedia.org/wiki/Prolog) is a logic programming language. It works in such a way that you establish facts and rules, and then you are able to ask whether certain relationships can be established between atoms. This makes certain types of problems far easier to solve than if you were to implement a solution in _e.g._ JavaScript. Conversely, it makes solving other kinds of computations relatively hard. In the book _Seven Languages in Seven Weeks_, Prolog was compared to Dustin Hoffman's character in _Rainman_ — working in Prolog is a bit like communicating with an autistic savant.

The answers are typically `yes` or `no`.

If you have familiarity with Prolog, this joke is quite funny.

File: readme.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (styfle) [reply readme.md]: I'll defer to @wesbos on this one

**Pull request merged**